lib/ma_crossover/meta/validate_params.js

```diff
@@ -41,7 +41,7 @@ const validateParams = (args = {}) => {
   } = args
 
   if (!_includes(ORDER_TYPES, orderType)) return `Invalid order type: ${orderType}`
-  if (!_isFinite(amount)) return 'Invalid amount'
+  if (!_isFinite(amount) || amount === 0) return 'Invalid amount'
```

Comment:
I think there should be a check: `amount > 0`

Reply:
so before we validate the params .. we are processing them first which negates the value if its a sell order .. so a check for amount equals 0 is fine here

Reply:
I see. You are right